dist/cjs/src/luno.js

```diff
@@ -108,33 +108,35 @@ class luno extends luno$1 {
                         'funding_address': 1,
                         'listorders': 1,
                         'listtrades': 1,
+                        'send_fee': 1,
```

Comment:
@DelAnt sorry I did not notice you added changes to this file `dist/cjs/src/luno.js`, this is automatically generated, you have to add them to `ts/src/luno.ts` instead

Reply:
Sorry. I should have spotted that as well. Does this work now?

Reply:
@DelAnt yeah it's ok now (I restored the other file)